libs/constants/src/lib/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const MAX_ERR_LOGS_COUNT = 1024;
```

Comment:
Add descriptions for each constant; together with some context to where it is being used.

Reply:
Added descriptions 93015d626ca686a6707ee4f18bf61e385507952b